ros/cpp/src/ptg_datahub/src/ptg_datahub.cpp

```diff
@@ -384,6 +581,7 @@ int main(int argc, char * argv[])
   t8.join();
   t9.join();
   t10.join();
+  t11.join();
```

Comment:
This is fine for now, but it might be better to start/join these threads as part of the node-class operation, e.g. start in constructor, join in destructor.